### PR TITLE
T28444 : Test für QPS bauen

### PR DESCRIPTION
**Pull request by typ1804**

### CODE REVIEW

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php [hunk 1]
```diff
@@ -90,6 +90,14 @@ public function saveAction()
         $this->_redirect('*/*/index');
     }
 
+    public function loadAction()
+    {
+        $cron = new Mageone_Qps_Model_Cron();
```
Comment (Schrank) [inline src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php]: Mage::getModel()

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php [hunk 2]
```diff
@@ -90,6 +90,14 @@ public function saveAction()
...
+        $cron->getRules();
```
Comment (Schrank) [inline src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:96]: Can you catch the exception here and if we are in admin context (not sure how to test) `Mage::getSession('admin/session')->addWarning(...)`

not 100% sure about the code, please check :D

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php [hunk 3]
```diff
@@ -12,7 +12,7 @@ public function indexAction(): void
 
     public function editAction()
     {
-        $id    = $this->getRequest()->getParam('id');
+        $id = $this->getRequest()->getParam('id');
```
Comment (Schrank) [inline src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:15]: be careful with these whitespace-only changes. normally no problem, but if customers want to review the changes, it is harder.

**Review by Schrank — CHANGES_REQUESTED** [review]

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php (hunk 3 above)
Comment (typ1804) [reply src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:15]: This kind of white space shouldn't even happen. It’s just bad. All unnecessary white space is.

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php (hunk 2 above)
Comment (typ1804) [reply src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:96]: There is no exception to catch. It's already caught in the getRules-function

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php (hunk 1 above)
Comment (typ1804) [reply src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php]: done

File: src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php (hunk 3 above)
Comment (Schrank) [reply src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php:15]: I'm happy to challenge that opinion. With a beer (or something else) on a nice evening :-)

**Review by Schrank — APPROVED** [review]

**Pull request merged**